server/world/sleep.go

```diff
@@ -0,0 +1,52 @@
+package world
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/google/uuid"
+)
+
+// Sleeper represents an entity that can sleep.
+type Sleeper interface {
+	Entity
+
+	Name() string
+	UUID() uuid.UUID
+
```

Comment:
I couldn't see any usage of these methods. Try removing any methods from interfaces that aren't used.

Reply:
![{4CD39432-01BB-40D1-BABB-28279562CF7A}](https://github.com/user-attachments/assets/9cecdb58-125f-4d5c-a77e-0fb672e0177c)
![{E97426C8-1488-4E8B-8449-BE6A16C17F29}](https://github.com/user-attachments/assets/673e90e0-2390-48a4-b516-85f774a949ac)
but these methods are used

Reply:
Good point! Thanks for clarifying.